cmake/DD4hepBuild.cmake

```diff
@@ -43,6 +43,11 @@ macro(dd4hep_set_compiler_flags)
     set ( COMPILER_FLAGS ${COMPILER_FLAGS} -Winconsistent-missing-override -Wno-c++1z-extensions -Wheader-hygiene )
   endif()
 
+  IF( DD4HEP_USE_GEANT4_UNITS )
+    MESSAGE(STATUS "+++ Building DD4hep with Geant4 units......")
+    add_compile_definitions(DD4HEP_USE_GEANT4_UNITS=1)
```

Comment:
This should just be added as a `target_compile_definitions` to DDG4(?)

Reply:
Maybe.
Then it also must enter in cmake/DD4hepConfig.cmake.in because DD4hepUnits.h must be consistently used for all
depending code.

Reply:
Which it would because of the transitivity of the cmake target properties.

Reply:
I think I misunderstood your comment.
The flag DD4HEP_USE_GEANT4_UNITS has nothing to do with Geant4.
It tells dd4hep to instruct TGeo to natively compute all quantities in the same units as Geant4 has (mm,ns,MeV).
This is independent of DDG4. I will try to do the same thing as with DD4HEP_DEBUG